webpack/assets/javascripts/react_app/components/Bookmarks/BookmarksReducer.js

```diff
@@ -48,8 +43,6 @@ export default (state = initialState, { type, payload, response }) => {
           )
         )
         .set('showModal', false);
```

Comment:
would this `showModal` affect the component if we are using the modal infrastructure?

Reply:
@laviro Good catch, this is no longer needed